gulp/tasks/prepare.js

```diff
@@ -1,12 +1,15 @@
 /* Environment */
-const DEVELOPMENT = require('../environment').isDevelopment;
+const config = require('../config');
+const DEVELOPMENT = config.environment.isDevelopment;
 
-/* Plugins */
+/* Gulp */
 const gulp = require('gulp');
+
+/* Plugins */
 const runSequence = require('run-sequence');
 
 // TODO run browserSync after all tasks finished
-const devSequence = ['clean', ['icon', 'styles', 'js'], 'tpl', 'styleguide', 'copySgAssets'];
+const devSequence = ['clean', 'images', 'svg', ['styles', 'js', 'tpl'], 'styleguide'];
```

Comment:
Where is `copySgAssets`? `images` and `svg` run in parallel with `styles` and `js`. `tpl` must be after `styles`.